### PR TITLE
Dockerfy

### PR DESCRIPTION
**Pull request by kevindoveton**
dockerfy it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+docker-compose.yml
```
Comment (lewisdaly) [inline .gitignore:2]: We shouldn't ignore it!

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+FROM node:latest
```
Comment (lewisdaly) [inline Dockerfile:1]: Probably best to lock down a version, but it doesn't really matter

File: .gitignore (hunk 1 above)
Comment (kevindoveton) [reply .gitignore:2]: It contains the token though?

Comment (lewisdaly) [reply .gitignore:2]: Nah, don't put the token in there. Always pull the token from an environment variable, and add it to an `env.sh` script that sets up the environment. `env.sh` can do some fancy stuff like download the token from a secure place with aws credentials, or look for a special file that we don't commit to the repo.

File: processes.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+	"apps" : [
```
Comment (lewisdaly) [inline processes.json:2]: This is cool. Never seen that before!

**Comment by lewisdaly** [comment]
I'll just merge it in and write some scripts to set up the access token.

**Pull request merged**